src/tray.rs

```diff
@@ -1,11 +1,53 @@
+use std::io::Cursor;
+
+use image::Rgba;
 use ksni;
 
 #[derive(Debug)]
-struct Tray;
+struct Tray {
+    icon: ksni::Icon,
+}
+
+impl Tray {
+    pub fn new() -> Self {
+        let rgba_image = {
+            let cursor = {
+                let image_bytes = include_bytes!("../assets/flakeshot_logo_dpi_96.png");
+                Cursor::new(image_bytes.to_owned())
+            };
+            image::io::Reader::with_format(cursor, image::ImageFormat::Png)
+                .decode()
+                .unwrap()
```

Comment:
except

Reply:
Hm... I added the following change:

I move this whole block into an external function since the converted image will be in the binary anyhow and added a test which always make sure that the function, which returns the system-tray-image, will always work.

What do you think about it?